public/css/latest-breach.css

```diff
@@ -1,13 +1,12 @@
 
 .latest-breach-wrapper {
+  background-color: rgba(255, 255, 255, 1);
   overflow: hidden;
   flex-direction: column;
   margin: auto;
-  box-shadow: 1px 1px 10px #8181810e;
-  border-radius: var(--radius);
+  box-shadow: 1px 1px 10px #4444440e;
```

Comment:
Not sure if we want to convert this to `rgba()` syntax, I think this may be the only `#rrggbbaa` format.